controller/usercontroller.js

```diff
@@ -1,96 +1,24 @@
-import express from 'express';
-import mongoose from 'mongoose';
-import jwt from 'jsonwebtoken';
-import user from '../Model/user';
-
-
-class usercontroller {
-    static createUser(req, res) {
-        if (!req.body.email) {
-            return res.status(400).send({
-                status: 400,
-                message: 'Email is required',
-            });
-        } if (!req.body.firstName) {
-            return res.status(400).send({
-                status: 400,
-                message: 'First Name is required',
-            });
-        } if (!req.body.lastName) {
-            return res.status(400).send({
-                status: 400,
-                message: 'Last Name is required',
-            });
-        } if (!req.body.password || req.body.password.length < 7) {
-            return res.status(400).send({
-                status: 400,
-                message: 'password  is required and should be a minimum of 8charachers',
-            });
-        }
-        user.findOne({ email: req.body.email }).exec((err, nuser) => {
-            if (!nuser) {
-
-                const instance = new user({                    
-                    firstName: req.body.firstName,
-                    lastName: req.body.lastName,
-                    email: req.body.email,
-                    password: req.body.password
-                });
-                jwt.sign({ instance }, 'protect', { expiresIn: '1h' }, (err, token) => {
-                    if (err) { console.log(err) }
-                    const tk = token.split(',')
-                  res.json(tk);
-                });
-                try {
-                    const a = instance.save((err) => {
-                        if (err) throw err;
-
-                        res.json({
-                            status: 201,
-                            message: 'Account created!',
-                            data: instance
-                        });
-                    });
-                } catch (e) {
-
-                }
-
-            } else {
-                res.status(400).send({ message: 'Failed! Email is already in exist!' });
-
-            }
-        });
-
-
-
-
 
+import User from '../services/userservices';
+import helper from '../helper/password';
+import helperlogin from '../helper/login';
+export default class UserController {
+    static async signup(req, res) {
+      const inputFormData = req.body;
+      const textPassword = inputFormData.password;
+      inputFormData.password = helper.enclippass(textPassword);
+      const user = await User.createUser(inputFormData);
+     return res.json({
+         status:201,
+         message:"Acount created",
+         data:user
```

Comment:
You are not supposed to return the user upon signup and login, you are supposed to return `message and token` only.